core/src/main/resources/org/elasticsearch/plugins/plugin-install.help

```diff
@@ -20,6 +20,8 @@ DESCRIPTION
 
     The notation of 'username/repository' refers to a github repository.
 
+    The argument can be an valid <url> which points to a download or file location for the plugin to be loaded from.
+
 EXAMPLES
 
     plugin install elasticsearch-analysis-kuromoji
```

Comment:
If your change is not related to the issue @spinscale [found](https://github.com/elastic/elasticsearch/pull/12775/files#r37058692), then yes we should fix that in another issue.